### PR TITLE
feat: rule dev and updates

### PR DESCRIPTION
**Pull request by nasbench**

### CODE REVIEW

File: rules/proxy/proxy_exchange_owassrf_exploitation.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - attack.initial_access
+    - attack.t1190
+logsource:
+    category: proxy
+detection:
+    selection:
+        # Look for the header: X-OWA-ExplicitLogonUser: owa/mastermailbox@outlook.com
+        cs-method: 'POST'
+        c-uri|contains|all:
+            - '/owa/mastermailbox'
```
Comment (frack113) [inline rules/proxy/proxy_exchange_owassrf_exploitation.yml:20]:
```suggestion
            - '/owa/'
```
the generic payload : `https://{exchange_host}/owa/{email_address]/powershell`

Comment (frack113) [reply rules/proxy/proxy_exchange_owassrf_exploitation.yml:20]: can add a `@` for the email

File: rules/web/web_exchange_owassrf_exploitation.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    - attack.initial_access
+    - attack.t1190
+logsource:
+    category: webserver
+detection:
+    selection:
+        # Look for the header: X-OWA-ExplicitLogonUser: owa/mastermailbox@outlook.com
+        cs-method: 'POST'
+        c-uri|contains|all:
+            - '/owa/mastermailbox'
```
Comment (frack113) [inline rules/web/web_exchange_owassrf_exploitation.yml:20]:
```suggestion
            - '/owa/'
            - `@`
```

File: rules/proxy/proxy_exchange_owassrf_exploitation.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+logsource:
+    category: proxy
+detection:
+    selection:
+        cs-method: 'POST'
+        sc-status: 200
+        c-uri|contains|all:
+            - '/owa/'
+            - '@'
+            - '/powershell'
```
Comment (Neo23x0) [inline rules/proxy/proxy_exchange_owassrf_exploitation.yml:22]: I'm not sure how the proxy would print the `@` symbol to the log. Maybe it uses the URL encoded version. 
```suggestion
        c-uri|contains|all:
            - '/owa/'
            - '/powershell'
        c-uri|contains:
            - '@'
            - '%40'
```

**Pull request merged**